### PR TITLE
fix: strip trailing semicolon on sql runners

### PR DESCRIPTION
**Pull request by owlas**
Closes: #11181

### CODE REVIEW

File: packages/backend/src/queryBuilder.ts [hunk 1]
```diff
@@ -307,7 +307,10 @@ export const applyLimitToSqlQuery = ({
     limit: number | undefined;
 }): string => {
     if (limit === undefined) return sqlQuery;
-    return `WITH user_sql AS (${sqlQuery}) select * from user_sql limit ${limit}`;
+    return `WITH user_sql AS (\n${sqlQuery.replace(
+        /;$/,
```
Comment (ZeRego) [inline packages/backend/src/queryBuilder.ts:311]: We need the multi-line option to capture the end of each line

```suggestion
        /;$/m,
```

Test with
```
with test as (SELECT * FROM "postgres"."jaffle"."customers")

select * from test;

-- test "postgres"."public"."dashboard_tile_comments"
```

Comment (ZeRego) [reply packages/backend/src/queryBuilder.ts:311]: We are not handling the edge case for now

> If we start stripping semicolons anywhere else but the last character, it might be in a string or something and not be a terminating semicolon

**Review by ZeRego — APPROVED** [review]
